server/tests/replicated_loglet.rs

```diff
@@ -188,7 +188,7 @@ mod tests {
         .await
     }
 
-    #[restate_core::test(flavor = "multi_thread", worker_threads = 4)]
+    #[test(restate_core::test(flavor = "multi_thread", worker_threads = 4))]
```

Comment:
is this for logging?

Reply:
Yes, exactly. It helps me when debugging tests locally.